### PR TITLE
Updates to the Dremio Setup page

### PR DESCRIPTION
**Pull request by rheath-dremio**
## Description & motivation
We've updated Dremio's support for dbt.
Please do not merge, yet.

### CODE REVIEW

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ meta:
   authors: 'Dremio (formerly Fabrice Etanchaud)'
   github_repo: 'dremio/dbt-dremio'
   pypi_package: 'dbt-dremio'
-  min_core_version: 'v1.1.0'
+  min_core_version: 'v1.3.0'
   cloud_support: Not Supported
```
Comment (jlarue26) [inline website/docs/reference/warehouse-setups/dremio-setup.md]: This is referring to dbt Cloud? If so you are correct - not supported

**Review by jlarue26 — COMMENTED** [review]
looks good @rheath-dremio . One minor question

File: website/docs/reference/warehouse-setups/dremio-setup.md (hunk 1 above)
Comment (rheath-dremio) [reply website/docs/reference/warehouse-setups/dremio-setup.md]: Yes, that refers to dbt Cloud. Thanks.

**Comment by jlarue26** [comment]
@runleonarun @dataders we are ready for this documentation to be published.
Are you able to approve and merge the PR please?

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ meta:
   authors: 'Dremio (formerly Fabrice Etanchaud)'
   github_repo: 'dremio/dbt-dremio'
   pypi_package: 'dbt-dremio'
-  min_core_version: 'v1.1.0'
+  min_core_version: 'v1.3.0'
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/dremio-setup.md]: to me, `min_core_version` means the oldest version of dbt-core with which the adapter is compatible. so i interpret this change to mean that there is no version of dbt-dremio that is compatible with `dbt-core==1.1.0`, only from `1.3.0= and onwards...

File: website/docs/reference/warehouse-setups/dremio-setup.md [hunk 3]
```diff
@@ -76,11 +79,6 @@ Before connecting from project to Dremio Cloud, follow these prerequisite steps:
     * `dremio_cloud` for working with Dremio Cloud
     * `software_with_username_password` for working with a Dremio Software cluster and authenticating to the cluster with a username and a password
     * `software_with_pat` for working with a Dremio Software cluster and authenticating to the cluster with a personal access token
-4. Append these lines to the end of the content of the `dbt_project.yml` file at the root of your project directory:
-```
-vars:
-    dremio:reflections_enabled: false
-```
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/dremio-setup.md:83]: is this setting no longer valid?

**Review by dataders — DISMISSED** [review]
tiny little notes here!

File: website/docs/reference/warehouse-setups/dremio-setup.md (hunk 3 above)
Comment (jlarue26) [reply website/docs/reference/warehouse-setups/dremio-setup.md:83]: It is no longer necessary and we changed default behaviour from being reflections enabled to now being false. The reason for this change is that you need to be an administrator in Dremio to enable reflections.

We are working on supplementary documentation to explain how reflections work with dbt and are targeting for publishing this next week.

The supplementary documentation is based on the README from the community connector that has been tested for accuracy and currency, and cleaned up fore readability.

File: website/docs/reference/warehouse-setups/dremio-setup.md (hunk 2 above)
Comment (jlarue26) [reply website/docs/reference/warehouse-setups/dremio-setup.md]: Gotcha.

We should have min version being v1.1.0 and will update. We do support dbt-core v1.1.0, but do not support any earlier versions. Fabrice's adapter would have be used for those.

Comment (jlarue26) [inline website/docs/reference/warehouse-setups/dremio-setup.md]:
```suggestion
  min_core_version: 'v1.1.0'
```

**Review by dataders — APPROVED** [review]
:shipit:

**Pull request merged**